### PR TITLE
Update 00-spectro-k8s-dashboard.md

### PR DESCRIPTION
**Pull request by arvind-sys**
This PR adds new content to the Kubernetes Dashboard Pack related to RBAC. 
💻 [Preview](https://deploy-preview-1161--docs-spectrocloud.netlify.app/)
 * [Kubernetes Dashboard Pack](https://deploy-preview-1161--docs-spectrocloud.netlify.app/integrations/spectro-k8s-dashboard)
 * [Cluster Management RBAC]( https://deploy-preview-1161--docs-spectrocloud.netlify.app/clusters/cluster-management/cluster-rbac/#userbacwithoidc)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1161#" title="Author self-approved">arvind-sys</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @arvind-sys. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md [hunk 1]
```diff
@@ -13,197 +13,264 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
+  metadata:
+    namespace: default
+    name: pod-reader
+  rules:
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["get", "watch", "list"]
+  ```
+
+- **Cluster Role** An entity that is assigned a set of access permissions scoped to the cluster and all of its Kubernetes namespace. ClusterRoles do not have a namespace assigned.
```
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:39]:
```suggestion
- **Cluster Role** An entity that is assigned a set of access permissions scoped to the cluster and all of its Kubernetes namespaces. ClusterRoles do not have a namespace assigned.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md [hunk 2]
```diff
@@ -13,197 +13,264 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
+<br />
+
+<WarningBox>
+
+Palette does not provide a way for roles to be configured natively through its platform. You can create roles by using a manifest layer in the cluster profile. RBAC management only allows you to specify role bindings. 
+
+</WarningBox>
+
+
+Use the steps below to create a roleBinding or ClusterRoleBinding for your host clusters.
```
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:93]:
```suggestion
Use the steps below to create a RoleBinding or ClusterRoleBinding for your host clusters.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md [hunk 3]
```diff
@@ -13,197 +13,264 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
+
+
+<br />
+
+# Palette Roles and Kubernetes Roles
+
+Palette offers a set of [default roles](/user-management/palette-rbac#palettespecific(default)roles:) you can assign to your users. The Palette roles are only in scope at the platform level. This means you can manage the permissions for users' actions in Palette, such as creating or deleting clusters, creating projects, creating users, etc.
```
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:100]:
```suggestion
Palette offers a set of [default roles](/user-management/palette-rbac#palettespecific(default)roles:) you can assign to your users. The Palette roles are only in scope at the platform level. This means you can manage the permissions for users' actions in Palette, such as creating or deleting clusters, creating projects, creating users, and more.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md [hunk 4]
```diff
@@ -13,197 +13,264 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
+
+To create or modify a role binding for an active cluster. Navigate to the cluster details page and click on **Settings**. Select **RBAC** from the left **Settings Menu**. 
+
+![A view of the cluster settings page for an active cluster](/clusters_cluster-management_cluster-rbac_cluster-settings.png)
+
+
+The RBAC settings view contains two tabs:
+
+* **Cluster** Use this tab to create a ClusterRoleBinds.
+* **Namespaces** Use this tab to create a RoleBinding within Kubernetes namespaces.
```
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:137]:
```suggestion
* **Namespaces**: Use this tab to create a RoleBinding within Kubernetes namespaces.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md [hunk 5]
```diff
@@ -13,197 +13,264 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
+![A view of the cluster settings page when creating a cluster](/clusters_cluster-management_cluster-rbac_cluster-creation-settings.png)
+
+To create or modify a role binding for an active cluster. Navigate to the cluster details page and click on **Settings**. Select **RBAC** from the left **Settings Menu**. 
+
+![A view of the cluster settings page for an active cluster](/clusters_cluster-management_cluster-rbac_cluster-settings.png)
+
+
+The RBAC settings view contains two tabs:
+
+* **Cluster** Use this tab to create a ClusterRoleBinds.
```
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:136]:
```suggestion
* **Cluster**: Use this tab to create a ClusterRoleBinds.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md [hunk 6]
```diff
@@ -13,197 +13,264 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
+Select the tab for your specific role scope to learn how to create the appropriate binding.
+
+<Tabs>
+<Tabs.TabPane tab="Assign a Cluster Role" key="clusterRoleBinding">
 
-**Role** sets access permissions within a namespace. During Role creation, the namespace it belongs to needs to be specified.
+1. From the cluster settings view, select the **RBAC** tab.
+
+
+2. Click on **Add New Binding**
```
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:147]:
```suggestion
2. Click on **Add New Binding**.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md [hunk 7]
```diff
@@ -13,197 +13,264 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
+
+
+3. Fill out the following details:
+  * Role Name: Define a custom role name to identify the cluster role
```
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:151]:
```suggestion
  * Role Name: Define a custom role name to identify the cluster role.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md [hunk 8]
```diff
@@ -13,197 +13,264 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
+</Tabs.TabPane>
+
+<Tabs.TabPane tab="Assigne a Namespace Role" key="roleBinding">
+
+1. From the cluster settings view, select the **RBAC** tab.
 
-**ClusterRoleBinding** binds or associates a ClusterRole with a Subject (users, groups, or service accounts). ClusterRole and ClusterRoleBinding function similar Role and RoleBinding with a broader scope. ClusterRoleBinding grants access cluster-wide as well as multiple namespaces.
 
-# New clusters
+2. Click on **Add New Binding**
```
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:173]:
```suggestion
2. Click on **Add New Binding**.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md [hunk 9]
```diff
@@ -13,197 +13,264 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
 
-While configuring the cluster (Cluster Settings) during the cluster creation, the user can select RBAC from the left menu. There are two available options for setting up RBAC:
 
-* **Cluster** to create a RoleBinding with cluster-wide scope (ClusterRoleBinding).
-* **Namespaces** to create a RoleBinding within namespaces scope (RoleBinding).
-Palette users can choose role creation based on their resource requirements.
+3. Add the namespace name or provide a regular expression to automatically apply the following settings to other namespaces in the future. Ex: `/^web-app/`. Click on **Add To List**.
```
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:176]:
```suggestion
3. Add the namespace name or provide a regular expression to automatically apply the following settings to other namespaces in the future. Example: `/^web-app/`. Click on **Add To List**.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md [hunk 10]
```diff
@@ -13,197 +13,264 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
 
-## Configure cluster role bindings
 
-* Select Cluster Settings -> RBAC -> Cluster
-* Click on “Add new binding” to open the “Add Cluster Role Binding” wizard. Fill in the following details:
+4.  Allocate resources to the selected namespace. You can allocate the maximum CPU and Memory the role is allowed to consume from the listed namespaces.
+
+
+5. Click on **Add New Binding**
```
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:182]:
```suggestion
5. Click on **Add New Binding**.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md [hunk 11]
```diff
@@ -13,197 +13,264 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
+
+
+3. Fill out the following details:
```
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:185]:
```suggestion
6. Fill out the following details:
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md [hunk 12]
```diff
@@ -13,197 +13,264 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
+  * Namespace: Select the namespace.
+  * Role Type: The type of role. You can specify either a role or a cluster role.
   * Role Name: Define a custom role name to identify the cluster role
```
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:188]:
```suggestion
  * Role Name: Define a custom role name to identify the cluster role.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md [hunk 13]
```diff
@@ -13,197 +13,264 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
...
+```shell
+kubectl get role <yourRoleNameHere> --namespace <namespace> --show-kind --export
 ```
-## Example
+
+
+# Use RBAC With OIDC
+
+
+This section explains how RBAC can be used with OIDC providers. You can apply these steps to all the public cloud providers except [Azure-AKS](/clusters/public-cloud/azure/aks/#configureanazureactivedirectory) and [EKS](/integrations/oidc-eks/) clusters. Azure AKS and AWS EKS require different configurations.
```
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:240]:
```suggestion
This section explains how to use RBAC with OIDC providers. You can apply these steps to all the public cloud providers except [Azure-AKS](/clusters/public-cloud/azure/aks/#configureanazureactivedirectory) and [EKS](/integrations/oidc-eks/) clusters. Azure AKS and AWS EKS require different configurations. 
```

File: content/docs/06-integrations/00-kubernetes-dashboard.md [hunk 14]
```diff
@@ -28,7 +28,7 @@ Palette supports provisioning a [reverse proxy dashboard](/clusters/cluster-mana
 
 </InfoBox>
 
-An version of this pack is available, Spectro Kubernetes Dashboard, which requires no configuration when used with the default settings. To learn more, check out [Spectro Kubernetes Dashboard](/integrations/spectro-k8s-dashboard).
+A user-friendly version of this pack is available, Spectro Kubernetes Dashboard, which requires minimal configurations when used with the default settings. To learn more, check out [Spectro Kubernetes Dashboard](/integrations/spectro-k8s-dashboard).
```
Comment (ritawatson) [inline content/docs/06-integrations/00-kubernetes-dashboard.md:31]:
```suggestion
A user-friendly version of this pack is available, Spectro Kubernetes Dashboard, which requires minimal configuration when used with the default settings. To learn more, check out [Spectro Kubernetes Dashboard](/integrations/spectro-k8s-dashboard).

```

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md [hunk 15]
```diff
@@ -94,11 +98,23 @@ However, if you change **Access** to **Public** and your cluster is in a private
 
 #### Identity Provider 
 
+All IDP options below require you to map a set of users or groups to a Kubernetes RBAC role. There are two options you can use to get started the Kubernetes Dashboard and an IDP.
```
Comment (ritawatson) [inline content/docs/06-integrations/00-spectro-k8s-dashboard.md:101]:
```suggestion
All IDP options below require you to map a set of users or groups to a Kubernetes RBAC role. There are two options you can use to get started with the Kubernetes Dashboard and an IDP.
```

**Review by ritawatson — APPROVED** [review]
The added flow diagram is helpful. 

I noticed that sometimes roleBinding is used and sometimes RoleBinding. I started correcting this to be RoleBinding and thought instead I'd just point it out. You may want to search and correct for consistency. 

Other minor corrections are needed.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1161#" title="Author self-approved">arvind-sys</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1161#pullrequestreview-1334065763" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1161#" title="Author self-approved">arvind-sys</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1161#pullrequestreview-1335401084" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1161#pullrequestreview-1334065763" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@arvind-sys: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 2de5d6e1638a743c0b5d1c9b3b442def7a3aa611 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1161/librarium_build/1634259923464359936) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Pull request merged**